tools/ci_connector_ops/ci_connector_ops/pipelines/commands/groups/connectors.py

```diff
@@ -190,27 +177,13 @@ def test(
             ctx.obj["concurrency"],
             ctx.obj["execute_timeout"],
         )
-        update_commit_status_check(
-            ctx.obj["git_revision"],
-            "success",
-            ctx.obj["gha_workflow_run_url"],
-            GITHUB_GLOBAL_DESCRIPTION,
-            GITHUB_GLOBAL_CONTEXT,
-            should_send=ctx.obj.get("ci_context") == CIContext.PULL_REQUEST,
-            logger=logger,
-        )
-    except dagger.DaggerError as e:
+    except Exception as e:
```

Comment:
We now catch all kind of Exception to always report a `failure` if anything goes wrong. Otherwise the CI check will remain `pending`.